idaes/apps/grid_integration/multiperiod/price_taker_model.py

```diff
@@ -55,6 +55,27 @@
 _logger = idaeslog.getLogger(__name__)
 
 
+def _compute_sse(data, centroids, idx):
+    """
+    PRIVATE METHOD
```

Comment:
Now that you have moved this outside the class, it is actually a function rather than a method. The distinction is that a method is a member of (defined inside) a class (and thus normally has the `self` argument), where as a function is not owned by a class (e.g. defined at the module level like this, or defined inside a method or function).